src/components/AuthzProvider.tsx

```diff
@@ -53,17 +60,15 @@ export const AuthzProvider: React.FC<{ organizationId: string }> = (props) => {
       hasRole: (role: UserRoleType) => hasRole(role, roles),
       isSuperadmin,
       isOwner: hasRole(UserRoleType.OWNER, roles),
-      isAdmin: hasRole(UserRoleType.ADMIN, roles),
+      isAdmin: hasAdminPermissions,
```

Comment:
I made this change in prep for upcoming features (superadmin being able to manage all organizations, + deactivating them), so that if a user has a superadmin role, they have the ADMIN role for all other orgs, to be able to get into an org, and manage it without being needed to be added to that organization, went back and looked through the code, and figured this scenario is actually already handled. Happy to revert this change.

Reply:
Sounds good to me, reverted!